shared/actions/fs/platform-specific.desktop.js

```diff
@@ -167,21 +167,24 @@ function* fuseStatusSaga(): Saga.SagaGenerator<any, any> {
   const state: TypedState = yield Saga.select()
   const prevStatus = state.fs.fuseStatus
 
-  let status = yield Saga.call(RPCTypes.installFuseStatusRpcPromise, {bundleVersion: ''})
+  let status = yield* Saga.callPromise(RPCTypes.installFuseStatusRpcPromise, {bundleVersion: ''})
   if (isWindows && status.installStatus !== RPCTypes.installInstallStatus.installed) {
     // Check if another Dokan we didn't install mounted the filesystem
-    const kbfsMount = yield Saga.call(RPCTypes.kbfsMountGetCurrentMountDirRpcPromise)
+    const kbfsMount = yield* Saga.callPromise(RPCTypes.kbfsMountGetCurrentMountDirRpcPromise)
     if (kbfsMount && fs.existsSync(kbfsMount)) {
-      status.installStatus = RPCTypes.installInstallStatus.installed
-      status.installAction = RPCTypes.installInstallAction.none
-      status.kextStarted = true
+      status = {
```

Comment:
to be extra careful we mark the return values here as readonly so if you override the same data you need to make your own copy. i didn't validate this logic at all
@jzila